extensions/amp-story/0.1/amp-story.css

```diff
@@ -14,5 +14,459 @@
  * limitations under the License.
  */
 
+/** Common */
+amp-story, amp-story-page, amp-story-grid-layer {
+  contain: strict !important;
+  overflow: hidden !important;
+}
+
+/** Story level */
 amp-story {
+  height: 100% !important;
+  position: relative !important;
+  text-rendering: geometricPrecision !important;
+  user-select: none !important;
+  width: 100% !important;
+}
+
+html.i-amphtml-story-standalone,
+html[amp-story],
+html[📖],
+html.i-amphtml-story-standalone body,
+html[amp-story] body,
+html[📖] body {
+  height: 100% !important;
+  margin: 0 !important;
+  padding: 0 !important;
+  width: 100% !important;
+}
+
+html.i-amphtml-story-standalone body,
+html[amp-story] body,
+html[📖] body  {
+  display: grid !important;
+}
+
+amp-story[standalone],
+html[amp-story] body > amp-story,
+html[📖] body > amp-story {
+  align-self: center;
+  box-shadow: 2px 2px 20px rgba(0, 0, 0, 0.5);
+  height: 100% !important;
+  justify-self: center;
+  max-height: 736px;
+  max-width: 414px;
+}
+
+amp-story[standalone]:-webkit-full-screen,
```

Comment:
Combine all fullscreen declarations below?

Reply:
Fun fact: you can't :(

Some browsers find other browsers' vendor prefixes to be illegal selectors, and instead of dropping just that part of the compound selector, it drops the _whole thing_.  In other words, because `:-moz-full-screen` is present in part of the selector, Chrome would drop `amp-story[standalone]:-webkit-fullscreen, amp-story[standalone]:-moz-full-screen`

Reply:
In that case, you should check what our CSS compiler does. I believe it combines all such selectors automatically.